addons/ace_interact/XEH_postInitClient.sqf

```diff
@@ -8,6 +8,7 @@ if (!hasInterface) exitWith {};
 
 ["ace_arsenal_displayClosed", {
     EGVAR(sys_core,arsenalOpen) = false;
+    [] call DFUNC(monitorRadiosHandler);
```

Comment:
Leaving a note here: We have to think about API for this, copy-pasting those lines is far from ideal.